metadata.rb

```diff
@@ -23,12 +23,15 @@
   supports os
 end
 
-depends          'apt' # We recommend '>= 2.1.0'. See CHANGELOG.md for details
-depends          'chef_handler', '>= 1.2' # We recommend '~> 1.3' with Chef < 12. See CHANGELOG.md for details
-depends          'windows' # We recommend '< 1.39.0' if running Chef >= 12.6. See README.md for details
-depends          'yum', '>= 3.0' # Use '~> 3.0' with Chef < 12
-
-suggests         'sudo' # ~FC052
+depends    'chef_handler'
```

Comment:
@remeh any reason to remove the version constraint on `chef_handler`?

Reply:
Removing the version constraint bump to use the latest one `3.0.3` which fills the need of one card of our todo-list about having a better usage of the chef-handler dep.
To be more precautious I could add the constraint `~> 3.0` if we really want to be sure that we stay with 3.x but as chef-handler is now part of Chef, not sure it's needed.
Please lmk if it's a misunderstanding.